bin/stop_fe.sh

```diff
@@ -19,6 +19,43 @@
 curdir=`dirname "$0"`
 curdir=`cd "$curdir"; pwd`
 
+SIG=15
+TIME_OUT=60
+
+OPTS=$(getopt \
+  -n $0 \
+  -o gh \
+  -l 'graceful' \
+  -l 'timeout:' \
+  -l 'help' \
+  -- "$@")
+
+eval set -- "$OPTS"
+
+usage() {
+    echo "
+This script is used to stop BE process
+Usage:
+    sh stop_be.sh [option]
```

Comment:
it is fe.

and `sh` is not suggested in the script. turn to `bash` for more feature rich script. and make sure `chmod +x` to the script, so the script can be correctly run with the interpreter from the `shebang` line.